### ISSUE TITLE
timestamp comparison in client expiration logic 

### ISSUE THREAD

**Issue by rnbguy** [cosmos/ibc-rs#1239]
The following inequality should be `>=`

https://github.com/cosmos/ibc-rs/blob/2378cd4ba45094b8ed856ff7dba5f1d0882f59ae/ibc-clients/ics07-tendermint/src/client_state/validation.rs#L224-L225

Considering the [`is_within_trust_period`](https://github.com/informalsystems/tendermint-rs/blob/6399b5b747fc42330b30a51c931a3244002529c4/light-client-verifier/src/predicates.rs#L112-L116) logic from `tendermint-rs`.

To summarize, `tendermint-rs` expires a header at `trusted_header_time + trusting_period`. However, in `ibc-rs`, a header is active till  `trusted_header_time + trusting_period`.
